website/content/api-docs/secret/transit.mdx

```diff
@@ -664,6 +664,11 @@ This endpoint returns high-quality random bytes of the specified length.
 - `format` `(string: "base64")` – Specifies the output encoding. Valid options
   are `hex` or `base64`.
 
+- `source` `(string: "platform")` - Specifies the source of the requested entropy.
+  `platform`, the default, sources entropy from the platform's entropy source. 
```

Comment:
My understanding, and correct me if I'm wrong here, but there's a subtle distinction here between _random data_ (i.e., output of a RNG construction) and _entropy_ (which is slower, used to seed the RNG, and arguably of higher quality (e.g,. thermal noise measurements) -- but of limited size). While `/dev/urandom` is frequently used to seed an RNG (chaining), it internally has a RNG and should be considered the former, not the latter.

My  2c., but since we've so far (IMO correctly) called it "random bytes" or wording to that effect, I don't think we should give the impression that _we're_ maintaining an internal RNG construction of our own...  (that we're seeding from an external entropy source). Instead, we should clarify we simply ferry it from another RNG

(I guess you could argue `all` creates its own RNG, albeit with a trivial construction :D).

Reply:
That's quite fair, will change.